src/components/UiSelect/index.vue

```diff
@@ -374,6 +374,7 @@ export default {
 
       this.isOpen = true
       this.selectedIndex = this.flattenOptions.indexOf(this.currentItem)
+      this.$emit('is-dropdown-shown', this.isOpen)
```

Comment:
replace with on `close` and `open` events

Reply:
fixed